pyproject.toml

```diff
@@ -36,7 +36,13 @@ dependencies = [
 	'pydantic == 2.8.2',
 	'openai == 1.28.0',
 	'BeautifulSoup4',
-	'nltk'
+	'nltk',
+	'langchain-community',
```

Comment:
with that we're including all of langchain.. could you make these dependencies optional (so that not all users have to install it)?